Src/BootCamp.Chapter/Program.cs

```diff
@@ -11,6 +11,58 @@ public class Program
     {
         static void Main(string[] args)
         {
+            // part1 
+            var contents = File.ReadAllLines(@"C:\Users\roelof\source\repos\RoelofWobben\CSharp-From-Zero-To-Hero\Src\BootCamp.Chapter\Input/Balances.corrupted");
+            for (int i = 0; i < contents.Length; i++)
+            {
+                var repairedText = contents[i].ToString().Replace("_", "");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4967975)

Reply:
Hmm, can someone explain that to me. I use a try with a nested if then. Or is the try also count as a if ?

Reply:
solved